projects/netgrif-components-core/src/lib/task-content/services/task-content.service.ts

```diff
@@ -226,10 +225,13 @@ export abstract class TaskContentService implements OnDestroy {
         const frontendActions = chFields.taskId === this.task.stringId && chFields[TaskContentService.FRONTEND_ACTIONS_KEY];
 
         Object.keys(chFields).forEach(changedField => {
-            if (!!this.taskFieldsIndex[chFields.taskId] && !!this.taskFieldsIndex[chFields.taskId][changedField]) {
-                this.updateField(chFields, this.taskFieldsIndex[chFields.taskId][changedField], frontendActions);
+            if (!!this.taskFieldsIndex[chFields.taskId] && !!this.taskFieldsIndex[chFields.taskId].fields
+                && !!this.taskFieldsIndex[chFields.taskId].fields[changedField]) {
+                this.updateField(chFields, this.taskFieldsIndex[chFields.taskId].fields[changedField], frontendActions);
             } else if (this.isFieldInTaskRef(changedField)) {
-                this._taskDataReloadRequest$.next(frontendActions ? frontendActions : undefined);
+                const taskId = this.getReferencedTaskId(changedField);
+                if (!!taskId)
+                    this.updateReferencedField(chFields, this.taskFieldsIndex[taskId].fields[changedField], frontendActions);
```

Comment:
`this.taskFieldsIndex[taskId].fields` is not checked if the property exists as on line 228.

Reply:
Corrected